modules/acquia_cms_event/config/optional/core.entity_view_display.node.event.horizontal_card.yml

```diff
@@ -28,53 +28,61 @@ mode: horizontal_card
 content:
   body:
     type: text_summary_or_trimmed
-    weight: 2
-    region: content
     label: hidden
     settings:
       trim_length: 300
     third_party_settings: {  }
-  content_moderation_control:
-    weight: 0
+    weight: 2
```

Comment:
@rajeshreeputra We are updating configurations, but we are missing update hooks. Can we please add those as well ? (Same for all configurations).

Reply:
added update_hook